.github/workflows/integration.yml

```diff
@@ -38,6 +38,10 @@ jobs:
         run: |
           sudo tests/build.sh ${{ matrix.test_os }}
 
+      - name: Run container tests
+        run:
+          sudo just test-container
```

Comment:
Perhaps it may be nicer to invoke `sudo` within the justfile?  

I know it goes against best practice, though if someone must remember which commands require `sudo` and which ones don't, I know I'd certainly just default to sudoing everything.

Reply:
Ah, so this `test-container` entrypoint is expected to be ran both rootful and rootless.  Yeah, there's not really anything that can be done here.